tests/query/processors/test_uuid_column_processor.py

```diff
@@ -14,288 +14,11 @@
 from snuba.clickhouse.query import Query
 from snuba.query.processors.uuid_column_processor import (
     UUIDColumnProcessor,
-    BetterUUIDColumnProcessor,
     UUIDTranslationError,
 )
 from snuba.request.request_settings import HTTPRequestSettings
 
-
```

Comment:
You moved these tests to the BetterUUIDColumnProcessor test case right?

Reply:
Yeah, they're all equivalent